ush/update_input_nml.py

```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/env python3
 
+"""
+Set AQM-specific namelist settingSet AQM-specific namelist settingss
```

Comment:
This line was added twice.
```suggestion
Set AQM-specific namelist settings
```